bancorprotocol/carbon-app
# [Disposable] Reset budget on direction change

fix #1359

Reset budget when changing direction change

Test case : 
- Start disposable strategy
- Add budget
- Change from buy order to sell order <- budget should be reset
- Validate
- Edit price on the same strategy
- Add budget
- Change from sell order to buy order <- budget should be reset
